examples/cra/src/App.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import logo from './logo.svg'
+import './App.css'
+import { Box } from '@evilfactory/edan-core'
+
+function App() {
+  return (
+    <>
```

Comment:
Add emotion themeprovider and value of provider use from src/theme/index.js#default

Reply:
in `edan core` add `src/ThemeProvider/index.js`
```javascript
import { ThemeProvider as Provider } from 'emotion-theming';
import { default as defaultTheme } from '../theme';


export default function ThemeProvider({children, theme}){
  return <Provider theme={theme}>{children}</Provider>;
}

ThemeProvider.defaultProps = {
  theme: defaultTheme
};
// proptypes
```

in `cra`, implement ThemeProvider in `App.js`.

@kevanantha